Game/Source/GDKTestGyms/Private/NFRConstants.cpp

```diff
@@ -0,0 +1,87 @@
+#include "NFRConstants.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineUtils.h"
+#include "GDKTestGymsGameInstance.h"
+#include "Interop/SpatialWorkerFlags.h"
+
+DEFINE_LOG_CATEGORY(LogNFRConstants);
+
+UNFRConstants::UNFRConstants()
+{
+	
+}
+
+bool UNFRConstants::SamplesForFPSValid() const
+{
+	if (bFPSSamplingValid)
+	{
+		return true;
+	}
+	bool bIsValidNow = FDateTime::Now().GetTicks() > TimeToStartFPSSampling;
+	if (bIsValidNow)
+	{
+		bFPSSamplingValid = true;
+	}
+	return bIsValidNow;
+}
+
+void UNFRConstants::InitWithWorld(UWorld* World)
+{
+	TimeToStartFPSSampling = FDateTime::Now().GetTicks() + FTimespan::FromSeconds(120.0f).GetTicks();
+
+	USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(World->GetNetDriver());
+	// Server fps
```

Comment:
These overrides aren't actually used anywhere yet right?

Reply:
They aren't but just so they are available for when we want to (latency test)